lib/index.js

```diff
@@ -85,14 +96,18 @@ AxeBuilder.prototype.withTags = function (tags) {
 AxeBuilder.prototype.analyze = function(callback) {
 	var context = normalizeContext(this._includes, this._excludes),
     driver = this._driver,
-    options = this._options;
+    options = this._options,
+    config = this._config;
 
 	inject(driver, function() {
 		driver
-			.executeAsyncScript(function(context, options) {
+			.executeAsyncScript(function(context, options, config) {
 				/*global document, axe */
+        if (config !== null) {
```

Comment:
My IDE is being a turd and not registering changes to that whitespace no matter how I create it, weird. I'll try fixing it another way